packages/user-agent-parser/user-agent-parser.0.1.3/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "Issuu Raccoons <racoon@issuu.com>"
+synopsis: "OCaml implementation of the user agent parse rules of uap-core"
+homepage: "https://github.com/issuu/uap-ocaml"
+doc: "https://issuu.github.io/uap-ocaml/"
+dev-repo: "git+https://github.com/issuu/uap-ocaml.git"
+bug-reports: "https://github.com/issuu/uap-ocaml/issues"
+build: [["dune" "build" "-p" name "-j" jobs]]
+run-test: [["dune" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "ocaml" {>="4.05.0"}
+  "alcotest" {with-test & >= "0.8.3"}
+  "base" {>= "v0.11"}
+  "dune" {>="1.3"}
+  "ppx_deriving" {build}
+  "ppx_let" {build}
```

Comment:
One last thing: `ppx_let` and `ppx_deriving` are used by the library. The first may have or may introduce a runtime, the second has a runtime, so it is customary to no longer tag them as build dependency.

I will make the change myself in this PR and merge, but please make this change (as well as the others) also on the upstream repository.

Reply:
The corresponding changes [have now been made upstream](https://github.com/issuu/uap-ocaml/pull/6).